<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#1c0040</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>800</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item alignment="Qt::AlignHCenter">
           <widget class="QFrame" name="frame_6">
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>600</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>2500</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>400</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Montserrat SemiBold</family>
                 <pointsize>20</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(170, 85, 255)

</string>
               </property>
               <property name="text">
                <string>FACECOCK</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
              <widget class="QFrame" name="frame_4">
               <property name="minimumSize">
                <size>
                 <width>450</width>
                 <height>600</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>450</width>
                 <height>600</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border: 1px solid rgb(170, 85, 255) ;
	border-radius: 20px;
	padding: 0px</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item alignment="Qt::AlignHCenter">
                 <widget class="QFrame" name="frame_9">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>500</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">border: 0;
border-radius: 0
</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="minimumSize">
                      <size>
                       <width>350</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>350</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <family>Montserrat SemiBold</family>
                       <pointsize>16</pointsize>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(170, 85, 255)

</string>
                     </property>
                     <property name="text">
                      <string>CRIAR CONTA</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="maximumSize">
                      <size>
                       <width>400</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <family>Montserrat SemiBold</family>
                       <pointsize>10</pointsize>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(170, 85, 255)</string>
                     </property>
                     <property name="text">
                      <string>EMAIL</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="input_email">
                     <property name="minimumSize">
                      <size>
                       <width>350</width>
                       <height>35</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>350</width>
                       <height>35</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QLineEdit {
	border-radius: 5px;
	background-color: transparent;
	border: 1px solid rgb(170, 85, 255);
	padding: 5px;
	color: rgb(170, 85, 255)
}
</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="placeholderText">
                      <string>Ex: example@gmail.com</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_1">
                     <property name="maximumSize">
                      <size>
                       <width>400</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <family>Montserrat SemiBold</family>
                       <pointsize>10</pointsize>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(170, 85, 255)</string>
                     </property>
                     <property name="text">
                      <string>SENHA</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="input_password">
                     <property name="minimumSize">
                      <size>
                       <width>350</width>
                       <height>35</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>350</width>
                       <height>35</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QLineEdit {
	border-radius: 5px;
	background-color: transparent;
	border: 1px solid rgb(170, 85, 255);
	padding: 5px;
	color: rgb(170, 85, 255)
}
</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="placeholderText">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="maximumSize">
                      <size>
                       <width>400</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <family>Montserrat SemiBold</family>
                       <pointsize>10</pointsize>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(170, 85, 255)</string>
                     </property>
                     <property name="text">
                      <string>NOME DE USUÁRIO</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="input_username">
                     <property name="minimumSize">
                      <size>
                       <width>350</width>
                       <height>35</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>350</width>
                       <height>35</height>
                      </size>
                     </property>
                     <property name="layoutDirection">
                      <enum>Qt::LeftToRight</enum>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QLineEdit {
	border-radius: 5px;
	background-color: transparent;
	border: 1px solid rgb(170, 85, 255);
	padding: 5px;
	color: rgb(170, 85, 255)
}
</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="placeholderText">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_5">
                     <property name="maximumSize">
                      <size>
                       <width>400</width>
                       <height>20</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <family>Montserrat SemiBold</family>
                       <pointsize>10</pointsize>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(170, 85, 255)</string>
                     </property>
                     <property name="text">
                      <string>TELEFONE</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="input_phone">
                     <property name="minimumSize">
                      <size>
                       <width>350</width>
                       <height>35</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>350</width>
                       <height>35</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QLineEdit {
	border-radius: 5px;
	background-color: transparent;
	border: 1px solid rgb(170, 85, 255);
	padding: 5px;
	color: rgb(170, 85, 255)
}
</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="placeholderText">
                      <string>Ex: (XX) 9XXXX-XXXX</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item alignment="Qt::AlignHCenter">
                 <widget class="QPushButton" name="create_button">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>200</width>
                    <height>33</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Montserrat SemiBold</family>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
	border-radius: 5px;
	background-color: transparent;
	border: 0.8px solid rgb(170, 85, 255);
	color: rgb(170, 85, 255)
}

QPushButton:hover {
	background-color: rgb(170, 85, 255);
	color: #1c0040
}</string>
                  </property>
                  <property name="text">
                   <string>Enviar</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_8">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="login_page_button">
                  <property name="minimumSize">
                   <size>
                    <width>230</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>200</width>
                    <height>33</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
	border-radius: 5px;
	background-color: transparent;
	border: 0.8px solid rgb(170, 85, 255);
	color: rgb(170, 85, 255)
}

QPushButton:hover {
	background-color: rgb(170, 85, 255);
	color: #1c0040
}</string>
                  </property>
                  <property name="text">
                   <string>Já possui conta?</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
